Dockerfile

```diff
@@ -1,26 +1,26 @@
-FROM openshift/base-centos7
+FROM registry.access.redhat.com/ubi8
 
-RUN yum install -y centos-release-scl  && \
-    yum remove -y  gcc                 && \
-    yum install -y devtoolset-7-gcc devtoolset-7-gcc-c++  && \
-    yum install -y rpmdevtools            \
-                   git                    \
-                   openssl-devel          \
-                   libicu-devel           \
-                   python-devel           \
-                   systemtap-sdt-devel    \
-                   make              
-
-ENV PATH $PATH:/opt/rh/devtoolset-7/root/usr/bin
+COPY rpmdevtools/ /root/rpmdevtools/
+RUN pushd /root/rpmdevtools && yum -y --nogpgcheck localinstall *.rpm && popd
+RUN yum install -y git                            \
+                   gcc                            \
+                   gcc-c++                        \
+                   openssl-devel                  \
+                   libicu-devel                   \
+                   procps-ng                      \
+                   python2-devel                  \
+                   python3-devel                  \
```

Comment:
Why both Python 2 and 3? I haven't built Node 12 yet. Has it moved to Python 3?

Reply:
There is one script in V8 which I believe needs Python3. I'm not 100% about the details as this was something that Yaakov provided in his work. The goal is to migrate all the scrips to Python3 eventually so I think we'll have to live with Python2 for a while still.

Reply:
Extra information about python 2x support:
https://github.com/python/devguide/pull/344